src/include/rados.h

```diff
@@ -256,6 +256,9 @@ extern const char *ceph_osd_state_name(int s);
 	f(CACHE_PIN,	__CEPH_OSD_OP(WR, DATA, 36),	"cache-pin")        \
```

Comment:
should get rid of the 'cherry-picked from' in the commit message, since that ref won't exist after this one is merged